sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -769,6 +772,11 @@ class MLPClassifier(BaseMultilayerPerceptron, ClassifierMixin):
         (how many times each data point will be used), not the number of
         gradient steps.
 
+    max_fun : int, optional, default 200
+        Maximum number of function calls. The solver iterates until convergence
+        (determined by 'tol') or this number of function calls.
```

Comment:
maybe explain quickly why `max_fun` is different from `max_iter` and the relation `max_fun >= max_iter`